assignments/lambda-classes.js

```diff
@@ -1 +1,110 @@
 // CODE here for your Lambda Classes
+
+class Person {
+  constructor(element) {
+    this.name = element.name;
+    this.age = element.age;
+    this.location = element.location;
+    this.gender = element.gender;
+  }
+  speak() {
+    return `Hello my name is ${this.name}, I am from ${this.location}`;
+  }
+}
+
+class Instructor extends Person {
+  constructor(element) {
+    super(element);
+    this.specialty = element.specialty;
+    this.favLanguage = element.favLanguage;
+    this.catchPhrase = element.catchPhrase;
+  }
+
+  demo(subject) {
+    console.log(`Today we are learning about ${subject}`);
+  }
+
+  grade(student, subject) {
+    console.log(`${student.name} receives a perfect score on ${subject}`);
+  }
+}
+
+class Student extends Person {
+  constructor(element) {
+    super(element);
+    this.previousBackground = element.previousBackground;
+    this.className = element.className;
+    this.favSubjects = element.favSubjects;
+  }
+
+  listsSubjects() {
+    this.favSubjects.forEach(element => {
+      console.log(element);
+    });
+  }
+
+  PRAssignment(subject) {
+    console.log(`${student.name} has submitted a PR for ${subject}`);
+  }
+
+  sprintChallenge(subject) {
+    console.log(`${student.name} has begun sprint challenge on ${subject}`);
+  }
+}
+
+class ProjectManager extends Instructor {
+  constructor(element) {
+    super(element);
+    this.gradClassName = element.gradClassName;
+    this.favInstructor = element.favInstructor;
+  }
+
+  standUp(channel) {
+    console.log(`${name} announces to ${channel}, @channel standby times!`);
+  }
+
+  debugsCode(subject) {
```

Comment:
the instruction says: 
```
debugsCode a method that takes in a student object  and a subject 
```
but you console logged: `steve.debugsCode('#CSS');`

If you were to pass the whole studen'ts object, like `amber`, just like wha the instruction says, your code will look like this:
```
debugsCode(student, subject) {
    console.log(`${this.name} debugs ${student.name}'s code on ${subject}`);
  }

steve.debugsCode(amber, "CSS");  // Steve debugs Amber's code on CSS
```